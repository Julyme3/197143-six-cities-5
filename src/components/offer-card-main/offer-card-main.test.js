import React from "react";
import renderer from "react-test-renderer";
import OfferCardMain from "./offer-card-main";

jest.mock(`../offer-card/offer-card`, () => `OfferCard`);

test(`OfferCardMain render`, () => {
  const tree = renderer
    .create(
        <OfferCardMain
          className="near-places__card"
          classNameInner="near-places__image-wrapper"
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
