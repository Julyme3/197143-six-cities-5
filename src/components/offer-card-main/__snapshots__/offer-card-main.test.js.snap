// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferCardMain render 1`] = `
<OfferCard
  className="near-places__card cities__place-card"
  classNameInner="near-places__image-wrapper cities__image-wrapper"
/>
`;
